src/OrbitGl/Track.h

```diff
@@ -49,26 +46,23 @@ class Track : public Pickable, public std::enable_shared_from_this<Track> {
   explicit Track(TimeGraph* time_graph, CaptureData* capture_data);
   ~Track() override = default;
 
-  virtual void SetCaptureData(CaptureData* capture_data) { capture_data_ = capture_data; }
-
-  virtual void Draw(GlCanvas* canvas, PickingMode picking_mode, float z_offset = 0);
+  void Draw(GlCanvas* canvas, PickingMode picking_mode, float z_offset = 0) override;
 
-  virtual void UpdatePrimitives(uint64_t min_tick, uint64_t max_tick, PickingMode picking_mode,
-                                float z_offset = 0);
-
-  // Pickable
-  void OnPick(int x, int y) override;
-  void OnRelease() override;
+  void UpdatePrimitives(uint64_t min_tick, uint64_t max_tick, PickingMode picking_mode,
+                        float z_offset = 0) override;
   void OnDrag(int x, int y) override;
-  [[nodiscard]] bool Draggable() override { return true; }
+
+  virtual void SetCaptureData(CaptureData* capture_data) { capture_data_ = capture_data; }
 
   [[nodiscard]] virtual Type GetType() const = 0;
-  [[nodiscard]] virtual bool Movable() { return true; }
+  [[nodiscard]] virtual bool Movable() { return !pinned_; }
```

Comment:
Only a comment: This might be reversed depending on how we would implement pinning. (for example, divided the TimeGraph in 2 horizontally)